CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
--
+- **cf-icons:** Add piggy-bank-check and split icons
```

Comment:
You'll need to format this for the bot to ingest it as:

```
**cf-icons:** [MINOR] Add piggy-bank and split icons
```

Reply:
Do we document this syntax anywhere?

Reply:
In the contributing doc https://github.com/cfpb/capital-framework/blob/canary/CONTRIBUTING.md#changing-the-codebase

Reply:
Hrm, except the example linked to there [is wrong](https://github.com/cfpb/capital-framework/pull/291/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12) 😬

Reply:
Oh weird, it must have changed at one point. Want to update that to point to a more recent PR?

Reply:
Done! Merged it straight to `canary` 🔥 .